app/src/main/java/at/tugraz/ist/swe/note/NoteActivity.java

```diff
@@ -0,0 +1,155 @@
+package at.tugraz.ist.swe.note;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.widget.EditText;
+import android.widget.Toast;
+
+public class NoteActivity extends AppCompatActivity {
+
+    private Menu _menu;
```

Comment:
You are not consistent in your style. Why are you using the underline in front of the variable name? 
All four variables are private (although the two without a modifier are package-private)
We said we gonna use the Android Code Style conventions. 
None of these fields is declared in the right way, as you can see in https://source.android.com/setup/contribute/code-style#follow-field-naming-conventions